src/roiextractors/extractors/hdf5imagingextractor/hdf5imagingextractor.py

```diff
@@ -47,10 +47,13 @@ def __init__(
         self._channel_names = channel_names
 
         self._file = h5py.File(file_path, "r")
-        if "mov" in self._file.keys():
+        if mov_field in self._file.keys():
```

Comment:
The standard `Hdf5ImagingExtractor` wasn't actually useful for my case, but I saw this and fixed it anyway. Can splinter into a different PR if needed.